person_test.go

```diff
@@ -3,6 +3,8 @@ package faker
 import (
 	"strings"
 	"testing"
+
+	"github.com/jaswdr/faker/internal/test"
```

Comment:
We don't need an internal package, the current structure is intentionally simple and only one single deep level